app/mainwindow.cpp

```diff
@@ -927,6 +919,11 @@ void MainWindow::markdownConverterChanged()
     }
 }
 
+void MainWindow::editorFontChanged()
```

Comment:
Is this a left over from a different change? Seems not to be related to this pull request.